File: bindings/python/nnw/enums.py
from enum import IntEnum

class Activation(IntEnum):
    LINEAR = 0
    RELU = 1
    SIGMOID = 2
    TANH = 3

class Optimizer(IntEnum):
    SGD = 0
    ADAMW = 1

class Loss(IntEnum):
    MSE = 0

File: Runtime/Core/NN/Enums.cs
namespace NNW.Core.NN
{
    public enum Optimizer : int
    {
        SGD = 0,
        ADAMW = 1
    }

    public enum Loss : int
    {
        MSE = 0,
    }

    public enum Activation : int
    {
        LINEAR = 0,
        RELU = 1,
        SIGMOID = 2,
        TANH = 3
    }
}
